CMakeLists.txt

```diff
@@ -217,10 +253,15 @@ endif()
 
 install(DIRECTORY include/CL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp")
 install(DIRECTORY include/SYCL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp")
-install(DIRECTORY include/hipSYCL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp")
+install(DIRECTORY include/hipSYCL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp" PATTERN include/hipSYCL/compiler EXCLUDE)
 
+if(NOT WIN32)
 install(PROGRAMS bin/syclcc DESTINATION bin)
+else()
+install(PROGRAMS bin/syclcc-clang DESTINATION bin RENAME syclcc)
+endif()
 install(PROGRAMS bin/syclcc-clang DESTINATION bin)
+
```

Comment:
Can you comment on what is going on here with the `syclcc`'s?

Reply:
`syclcc` is a symlink to `syclcc-clang`, which is not too well supported on Windows.. CMake doesn't like copying it:
```
CMake Error at cmake_install.cmake:57 (file):
  file INSTALL cannot read symlink
  "D:/Programmierung/SYCL/hipSYCLWin/bin/syclcc" to duplicate at
  "D:/Programmierung/SYCL/hipSYCL_install/bin/syclcc": No such file or
  directory.
```

thus, I made CMake just copy `syclcc-clang` to `syclcc` in the install directory.